frontend/test/metabase/scenarios/collections/permissions.cy.spec.js

```diff
@@ -392,6 +397,14 @@ describe("collection permissions", () => {
                 });
               });
             });
+
+            describe("managing dashboard from the dashboard's edit menu", () => {
+              it("should not be offered to duplicate dashboard in collections they have `read` access to", () => {
+                cy.visit("/dashboard/1");
+                cy.icon("ellipsis").click();
+                cy.findByText("Duplicate").should("not.exist");
+              });
+            });
```

Comment:
@nemanjaglumac hey, please take a look how does it align with the current tests structure

I see tests are grouped by permission on the highest level. Like:

```
// some pseudo code

onlyOn("curate", () => {
   it("some collection page test")
   describe("managing dashboard")
   describe("managing question")
})

onlyOn("view", () => {
   it("some collection page test")
})
```

Here I'm just trying to mirror the same approach, so `onlyOn("view")` tests have their own `describe` blocks for dashboard and question pages. Does it make sense?

Reply:
Yes, that's what I was thinking about.
I know that we have some code duplication now and it makes more sense to extract that whole describe block (possibly into a separate file like you did originally). So, just wanted to let you know it was definitely good thinking.

We'll have to live with the duplication and a bit longer `permissions.cy.spec.js` file just a bit longer until some higher decisions on permissions testing are made.

Thank you for all the work so far!

Reply:
Glad to hear! Maybe it's worth to add a comment at the top of the test file saying that it covers permissions for different pages? I agree we better keep everyting in a single place until we're sure how to approach permission tests, but it's easy to get confused as I did `cause `scanarios` tests mirror our URLs. So if somebody is looking for permissions tests for the dashboard page, they would probably go to `scenarios/dashboard` folder

Reply:
Makes sense.
Feel free to add comment as you continue working on the issue and adding more tests.

Since everything's green now and you already have approvals on this PR, maybe just merge it as it is and add comments later?